nwb_conversion_tools/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py

```diff
@@ -20,16 +22,21 @@
 INSTALL_MESSAGE = "Please install lxml to use this interface!"
 
 
-def subset_shank_channels(recording_extractor: se.RecordingExtractor, xml_file_path: str) -> se.SubRecordingExtractor:
+def subset_shank_channels(
+    recording_extractor: Union[se.RecordingExtractor, si.BaseRecording], xml_file_path: str
+) -> si.BaseRecording:
     """Attempt to create a SubRecordingExtractor containing only channels related to neural data."""
+    if isinstance(recording_extractor, se.RecordingExtractor):
+        recording = si.core.old_api_utils.OldToNewRecording(recording_extractor)
```

Comment:
```suggestion
        recording = OldToNewRecording(oldapi_recording_extractor=recording_extractor)
```
Though it may seem silly for a function that only has a single input argument, using keywords is never a bad idea to prevent future errors in the off-chance that a future or past version of that function has a different argument added to the first position; we've been caught by this before.

Certain built-in functions forbid keywords entirely, like `some_dict.get()`, but most functions you find from our own ecosystems will support it.

Reply:
This makes sense to me.